src/screens/simulation/common.js

```diff
@@ -54,6 +54,33 @@ export const formatParametersForModel = ({ start_date, ...parameters }) =>
         start_time: 0,
     });
 
+export const formatRulesForModel = (rules = [], parameters) =>
+    rules.reduce((acc, rule) => {
+        if (percentFields.includes(rule.field)) {
+            acc.push({
+                ...rule,
+                value: round(rule.value / 100),
```

Comment:
I would not use `round` here

Reply:
I think it would be better to build the rules right from the beginning. i.e. in `extractRulesFromValues`
In other words, I would integrate the formatting directly during the extraction, unless you need the formatting function somewhere else in the code.